Mathlib/Topology/UniformSpace/Defs.lean

```diff
@@ -213,83 +212,80 @@ theorem subset_iterate_compRel {s t : SetRel α α} (h : idRel ⊆ s) (n : ℕ)
 def IsSymmetricRel (V : SetRel α α) : Prop :=
   Prod.swap ⁻¹' V = V
 
-set_option linter.deprecated false in
-@[deprecated (since := "2025-03-05")]
-alias SymmetricRel := IsSymmetricRel
-
 /-- The maximal symmetric relation contained in a given relation. -/
-@[deprecated SetRel.symmetrize (since := "2025-03-05")]
+@[deprecated SetRel.symmetrize (since := "2025-10-17")]
```

Comment:
Here and throughout the rest of the file, these declarations were deprecated with 2025-03-05 as the date in #23181, so I've changed those deprecation dates to 2025-10-17 to be consistent with the other deprecations added then: ([link to file in diff](https://github.com/leanprover-community/mathlib4/pull/23181/files#diff-e45c27272452cd32af00487bc82cc30f8aba9c55c6572c1acab383141c83c60a)).